.github/actions/tests-ac/action.yml

```diff
@@ -0,0 +1,23 @@
+name: Deploy doc
```

Comment:
The name doesn't seem to reflect what happens inside...

Reply:
Thx, forgot to modify it ...